test/cookbooks/acme_test/recipes/acme_dns_client.rb

```diff
@@ -0,0 +1,110 @@
+node.default['osl-acme']['pebble']['always_valid'] = false
+node.default['osl-acme']['pebble']['command'] = '/usr/local/bin/pebble -config /opt/pebble/test/config/pebble-config.json -dnsserver :8053'
+
+build_essential 'osl-acme-test' do
+  compile_time true
+end
+
+chef_gem 'bcrypt' do
+  compile_time true
+end
+
+#
+# Configure Bind
+#
+
+bind_service 'default' do
+  action [:create, :start]
+end
+
+cookbook_file '/etc/named.conf'
+
+directory '/var/named/master' do
+  recursive true
+end
+
+template '/var/named/master/db.example.org' do
+  source 'db.example.org.erb'
+  notifies :restart, 'bind_service[default]', :immediately
+end
```

Comment:
This should be a new recipe in `osl-dns` Can you please do that?

Reply:
So currently this is just entirely test data and only used in testing, I'm a bit confused why it needs to go in `osl-dns`. In production, we'll just add `_acme-challenge` records to the zonefile, right? Where else would the recipe be used?